db/seeds.rb

```diff
@@ -5,3 +5,1045 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+
+# [].tap do |array|
+#   100.times do |i|
+#     created_at = Time.current
+
+#     array << {
+#       title: "This is #{i+1} Article",
+#       text: "Hello This is #{i+1} Text",
+#       created_at: created_at,
+#       updated_at: created_at
+#     }
+#   end
+
+#   result = Article.insert_all array
+#   puts result.inspect
+
+# 4.times do |i|
+#   created_at = Time.current
+#   result = Article.insert_all(
+#     [
+#       {
+#         title: "This is #{i+1} Article hai",
+#         text: "This is #{i+1} Text",
+#         created_at: created_at,
+#         updated_at: created_at
+#       }
+#     ]
+#   )
+# end
+
+# puts result.inspect
+
+# puts Article.count
+
+result = Article.insert_all(
+  [
+    {"title":"Geological Engineer","text":"orci pede venenatis non sodales sed tincidunt eu felis fusce posuere felis sed lacus morbi sem mauris"},
+    {"title":"Registered Nurse","text":"vitae mattis nibh ligula nec sem duis aliquam convallis nunc proin at turpis a pede posuere"},
+    {"title":"Statistician III","text":"primis in faucibus orci luctus et ultrices posuere cubilia curae duis faucibus accumsan odio curabitur convallis duis consequat"},
+    {"title":"Web Developer III","text":"tempor convallis nulla neque libero convallis eget eleifend luctus ultricies eu nibh quisque id justo sit amet sapien"},
+    {"title":"Programmer Analyst IV","text":"sapien sapien non mi integer ac neque duis bibendum morbi"},
```

Comment:
@charchitk01 Its a bug for having not null issue. We will create articles one by one
Make changes like this 
```suggestion
Article.create(
  [
    {title: "Geological Engineer", text: "orci pede venenatis non sodales sed tincidunt eu felis fusce posuere felis sed lacus morbi sem mauris"},
    {title: "Registered Nurse", text: "vitae mattis nibh ligula nec sem duis aliquam convallis nunc proin at turpis a pede posuere" }
  ]
)
```

Reply:
okay sir